environments/base/default-config.libsonnet

```diff
@@ -10,6 +10,11 @@
     key: 'thanos.yaml',
   },
 
+  minio: {
```

Comment:
let's only have this in the dev config, nothing but the dev config actually uses minio in any way

Reply:
Done